prow/cmd/admission/main.go

```diff
@@ -57,6 +57,7 @@ func (o *options) parse(flags *flag.FlagSet, args []string) error {
 func main() {
 	o := parseOptions()
 	logrusutil.NewDefaultFieldsFormatter(nil, logrus.Fields{"component": "admission"})
```

Comment:
I think it might be simpler to change what this call does to also set the report caller as well.

Reply:
@stevekuznetsov what do you have in mind? Something like `logrusutil.init` that does both the formatter and the reporter?

Reply:
That would work, yep!